ros_ign_bridge/README.md

```diff
@@ -61,8 +62,8 @@ To run the following examples you will also need these ROS packages:
 * `rqt_image_view`
 
 The following Ignition dependencies are also needed:
-* `libignition-msgs4-dev`
-* `libignition-transport7-dev`
```

Comment:
How about documenting that we support either `msgs5` + `transport8` or `msgs4` + `transport7`?

Reply:
72ee52e